docs/standard/events/how-to-handle-multiple-events-using-event-properties.md

```diff
@@ -38,7 +38,7 @@ To use event properties, you define the event properties in the class that raise
  The following C# example implements the event properties `MouseDown` and `MouseUp`, using an <xref:System.ComponentModel.EventHandlerList> to store each event's delegate. The keywords of the event property constructs are in bold type.  
   
 > [!NOTE]
->  Event properties are not supported in Visual Basic.  
+>  Event properties are not supported in Visual Basic 2005.  
```

Comment:
Is this accurate? This implies that they were supported in Visual Basic 2003, and again in Visual Basic 2008. And unless I'm mistaken, I think that the Event...End Event syntax is an event property; I'm not sure when it was introduced, though.

Reply:
What to do then? Event properties were even supported in VB6.

Reply:
This note is not valuable. A better one should tell when it was introduced.